2023-06-06/keynote/README.md

```diff
@@ -0,0 +1,14 @@
+Date & Location
+---------------
+Online @ Youtube.
+
+How to run this
+---------------
+Using plain & old bundle exec:
+```bash
+  bundle exec ruby app.rb
```

Comment:
@samgooi4189 : if you have ruby 3.2.2 in your linux, this should work out of the box, plain & old ruby style.

I mean: considering you run bundle install.
Do you use any ruby version manager?
I have this setup with `rvm` on my machine.

Reply:
Maybe for next month we can have a `bin` folder with `copy`, 'setup', `startup` (or `run`), so the documentation steps to create a new meetup slides, setup, and run becomes simply:
1. Create a new meetup slide template: `bin/copy 2023-05-02 2023-06-06`
2. Install:  `bin/setup`
3. Run: `bin/startup`

Anyway... maybe next time.